require.js

```diff
@@ -8,10 +8,8 @@
   var cache = {};
   var aliases = {};
   var has = ({}).hasOwnProperty;
-
-  var endsWith = function(str, suffix) {
-    return str.indexOf(suffix, str.length - suffix.length) !== -1;
-  };
+  var extExp = /\.[^\/]+$/
+  var indexExp = /\/index(\.[^\/]+)?$/
```

Comment:
Going with only `page.tpl` instead of `page` breaks backwards compatibility. Definitely can't do that.